sdk/samples/core/mosquitto_multiple_rpc_sample.c

```diff
@@ -253,9 +253,10 @@ az_result mqtt_callback(az_mqtt5_connection* client, az_event event, void* callb
   az_app_log_callback(event.type, AZ_SPAN_FROM_STR("APP/callback"));
   switch (event.type)
   {
-    case AZ_MQTT5_EVENT_CONNECT_RSP:
+    case AZ_EVENT_MQTT5_CONNECTION_OPEN_IND:
     {
-      az_mqtt5_connack_data* connack_data = (az_mqtt5_connack_data*)event.data;
+      az_event* connack_event = (az_event*)event.data;
```

Comment:
In case there was ever an event of a different type that signifies that the connection has been opened. I realize that for this indicator it will always be because of a connack event but for other indicators it might be more than one event. I agree with changing it to just the data, it'll simplify things